api/services/SurveyResultService.js

```diff
@@ -0,0 +1,9 @@
+"use strict";
```

Comment:
Make sure you have pulled in the most recent changes, this file already exists and has several functions.